Skip Thai character test when libthai is unavailable.

Taken from upstream bug tracker:
https://gitlab.gnome.org/GNOME/pango/merge_requests/161
https://gitlab.gnome.org/GNOME/pango/commit/b73284747ac937e31dc9191f84ed6e24284c88ee

diff --git a/tests/test-break.c b/tests/test-break.c
index 8549b678bba69360d07dce3af21d915412f20d51..47fa3009165e19a331aa04a0df4351ae2323933b 100644
--- a/tests/test-break.c
+++ b/tests/test-break.c
@@ -305,6 +305,12 @@ main (int argc, char *argv[])
       if (!strstr (name, "break"))
         continue;
 
+#ifndef HAVE_LIBTHAI
+      /* four.break involves Thai, so only test it when we have libthai */
+      if (strstr (name, "four.break"))
+        continue;
+#endif
+
       path = g_strdup_printf ("/break/%s", name);
       g_test_add_data_func_full (path, g_test_build_filename (G_TEST_DIST, "breaks", name, NULL),
                                  test_break, g_free);
